src/System.Net.Security/src/System/Net/SecureProtocols/SslState.cs

```diff
@@ -1576,13 +1577,14 @@ private Framing DetectFraming(byte[] bytes, int length)
 
             int version = -1;
 
-            if ((bytes == null || bytes.Length == 0))
+            if ((bytes == null || bytes.Length <= 0))
```

Comment:
I don't agree: that's an int and can get negative. The original Assert was checking for > 0 and that is different from the current form.